explorer/interpreter/value.h

```diff
@@ -1188,21 +1188,31 @@ class StaticArrayType : public Value {
  public:
   // Constructs a statically-sized array type with the given element type and
   // size.
-  StaticArrayType(Nonnull<const Value*> element_type, size_t size)
+  StaticArrayType(Nonnull<const Value*> element_type, size_t size,
+                  bool implicit)
```

Comment:
Might it make sense to have a single `std::optional<size_t> size` parameter, and similarly combine the two data members?

Reply:
Yeah, I think this workaround is better, I'll work on this